apps/editors/tests/test_views.py

```diff
@@ -1288,15 +1292,12 @@ class TestUnlistedPendingQueue(TestPendingQueue):
 
     def setUp(self):
         super(TestUnlistedPendingQueue, self).setUp()
-        # These should be the only ones present in the queue.
-        self.expected_names = ['Pending One', 'Pending Two']
```

Comment:
Yeah, without rewriting the whole, the only alternative I saw was calling `self.get_expected_addons_by_names()` in each test instead of in `setUp()` once, which sounds bad.